lost/src/main/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImpl.java

```diff
@@ -68,11 +70,12 @@
 
 
   public boolean isConnecting() {
-    return connecting;
+    return connectState == ConnectState.CONNECTING;
   }
 
   public FusedLocationProviderApiImpl() {
     connectionCallbacks = new HashSet<>();
+    connectState = ConnectState.IDLE;
   }
 
   public void connect(Context context, LostApiClient.ConnectionCallbacks callbacks) {
```

Comment:
When `connect` is called we should update the `connectState` to `ConnectState.CONNECTING`